### ISSUE TITLE
Checkov 2.0.176 runner error with plan input

### ISSUE THREAD

**Issue by rbmarkh** [bridgecrewio/checkov#1265]
**Describe the bug**
Running with a JSON input produces:

```
Traceback (most recent call last):
  File "/usr/local/bin/checkov", line 8, in <module>
    exit(run())
  File "/usr/local/lib/python3.9/site-packages/checkov/main.py", line 123, in run
    scan_reports = runner_registry.run(external_checks_dir=external_checks_dir, files=args.file,
TypeError: run() got an unexpected keyword argument 'bc_integration'
```

**To Reproduce**
1. `checkov -f tfplan.json`, where tfplan.json is a TF plan file from terraform show

**Expected behavior**
2.0.175 works fine with exactly the same input.

**Additional context**
Looks like in `checkov/main.py' the bc_integration stuff needed removing in line 123 too, see :

https://github.com/bridgecrewio/checkov/commit/c9f0b8ea2db24ce87b0c2a6ccf11d10b3fd5806c

(apologies for not doing a pull request, super busy!)

**Comment by gattil** [comment]
I observed the same issue with version 2.0.177, but it was solved with release 2.0.178 (https://github.com/bridgecrewio/checkov/pull/1266)

**Comment by schosterbarak** [comment]
Thank you for helping ti verify @gattil
